package.json

```diff
@@ -28,7 +28,7 @@
     "@nextcloud/auth": "^2.2.1",
     "@nextcloud/axios": "^2.4.0",
     "@nextcloud/browser-storage": "^0.3.0",
-    "@nextcloud/capabilities": "^1.1.0",
+    "@nextcloud/capabilities": "1.2.0",
```

Comment:
I don't see a reason to freeze future dependabot bumps

Reply:
> this cleans up extraneous prod.dependencies from the lib (babel, jest, e.t.c)

I didn't understand.

It has no such dependencies, hasn't it?

Reply:
And 1.2.0 is the latest version

Reply:
> It has no such dependencies, hasn't it?

See here: https://github.com/nextcloud-libraries/nextcloud-capabilities/commit/185933db7fad4f26a499450b3c220c78041923e1

Yes, it's the latest ATM, I can set it back as `^1.2.0`.

Reply:
> Yes, it's the latest ATM, I can set it back as `^1.2.0`.

Let's do it, then auto update works. With `@nextcloud/vue` it makes sense because of a regression.